activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessRuntimeImpl.java

```diff
@@ -255,6 +255,18 @@ public ProcessInstance start(StartProcessPayload startProcessPayload) {
         return processInstanceConverter.from(this.createProcessInstanceBuilder(startProcessPayload).start());
     }
 
+    @Override
+    public ProcessInstance startCreatedProcess(String processInstanceId) {
+        org.activiti.engine.runtime.ProcessInstance internalProcessInstance = runtimeService
+                                                                                .createProcessInstanceQuery()
+                                                                                .processInstanceId(processInstanceId)
+                                                                                .singleResult();
+        if (internalProcessInstance == null) {
+            throw new NotFoundException("Unable to find process instance for the given id:'" + processInstanceId + "'");
+        }
+       return processInstanceConverter.from(runtimeService.startCreatedProcessInstance(internalProcessInstance));
```

Comment:
You also need to check if the user has the rights to start the process. See https://github.com/Activiti/Activiti/blob/develop/activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessRuntimeImpl.java#L198-L200.